content/installation/python/PythonInstall.md

```diff
@@ -0,0 +1,100 @@
+<!--
+title: "Python Agent Installation"
+description: "Installing the Python Agent"
+tags: "python agent installation"
+-->
+
+## Installation
+
+To install the Contrast agent into your Python application, you must complete the following steps.  
+
+1. Add the <i>contrast-agent-*.tar.gz</i> to the application's `requirements.txt`. (This is outlined in the <b>Setup</b> section below.) 
+2. Add the *contrast_security.yaml* file to the application's *config* directory. (This is outlined in the **Configuration** section below.)
+3. Run the Go Speedracer service as a standalone service on the same server as the application. (This is outlined in the section below to **Run the Service**.)
```

Comment:
Are we giving customers the service under the name of "Go Speedracer" or just "Contrast Service" as Ruby does?

Reply:
Good point, Alex. Contrast Service is correct.

Reply:
I will update